client-app/shared/account/components/product-card-reorder.vue

```diff
@@ -153,17 +176,22 @@
                       ? "9999+"
                       : productItem.availabilityData?.availableQuantity
                   }}
-                  in stock</span
+                  {{ $t("shared.account.reorder_info_popup.product_card.in_stock_suffix") }}</span
                 >
               </div>
               <div v-else-if="isOutOfStock" class="flex items-center">
-                <span class="text-[color:var(--color-danger)] text-xs pt-1 whitespace-nowrap">Out of stock</span>
+                <span
+                  class="text-[color:var(--color-danger)] text-xs pt-1 whitespace-nowrap"
+                  v-t="'shared.account.reorder_info_popup.product_card.out_of_stock_suffix'"
```

Comment:
Why key contains `suffix` if it is the whole phrase?

Reply:
Becouse it is  a child component for reorder-info popup